plugins/catalog-react/src/components/EntityPeekAheadPopover/CardActionComponents/UserCardActions.tsx

```diff
@@ -22,12 +23,7 @@ import { UserEntity } from '@backstage/catalog-model';
  *
  * @private
  */
-export const UserCardActions = ({ entity }: { entity: UserEntity }) => {
-  return (
-    <>
-      {entity.spec.profile?.email && (
-        <EmailCardAction email={entity.spec.profile.email} />
-      )}
-    </>
-  );
+export const UserCardActions = (props: { entity: UserEntity }) => {
+  const email = props.entity.spec.profile?.email;
+  return email ? <EmailCardAction email={email} /> : null;
```

Comment:
Seems like a duplication of the Group one but maybe we want to do something specific for groups/user entities later 🤷

Reply:
yeah could have been an `OrgCardAction = (props: { entity: UserEntity | GroupEntity }` but 🤷 it's internal so we can refactor at a later time if we want